os/drivers/lcd/Kconfig

```diff
@@ -119,6 +119,11 @@ config LCD_ST7701
 	---help---
 		Enable LCD Driver, ST7701 with mipi.
 
+config LCD_ST7701SN
```

Comment:
ST7701 and ST7701SN are different?

Reply:
st7701 -> 3.9 inch LCD
st7701sn -> 4.3 inch LCD